README.md

```diff
@@ -228,3 +228,23 @@ customElements.define('c-e', class extends HTMLElement {});
 // 'added first'
 // The document is walked to attempt upgrades.
 ```
+
+### Settings
+
+The polyfill provides a few settings to improve performance by tweaking behavior.
+These settings typically have correctness trade offs (noted below) and should be
+used with caution.
+
+* `customElements.noDocumentConstructionObserver`: Set this flag to true to
+prevent the polyfill from mutation observing and upgrading DOM as it is added.
+This provides a small performance improvement during document parsing.
+With this setting on, the polyfill will not upgrade parser generated elements
+defined after an element definition. This setting should be used in
```

Comment:
> With this setting on, the polyfill will not upgrade parser generated elements defined after an element definition.

Sort of related to the above comment, parser-generated elements are still upgraded when parsing is manually triggered by `innerHTML`, `insertAdjacentHTML`, etc. I think this is more like "the polyfill will not upgrade elements created when parsing the main document's HTML" or something like that.

Reply:
Done.